recipes/nbsphinx/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "nbsphinx" %}
+{% set version = "0.2.9" %}
+{% set md5 = "d5a80f5fc9a6f2128b6498c1f41dd079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
```

Comment:
Ok! As usual, commenting out first to see how it goes before committing to delete. Unlike pymc3, this time round if the tests pass, could you let me remove them from the recipe before merging?